actions/RestTestTakers.php

```diff
@@ -110,12 +110,12 @@ class RestTestTakers extends \tao_actions_CommonRestModule
      *         description="Password"
      *     ),
      *     @OA\Property(
-     *         property="uiLg",
+     *         property="userLanguage",
```

Comment:
It should be marked as required. What do you think?

Reply:
You're right, I'll update the documentation :+1: